### DISCUSSION TITLE
why is the "get" required?

### DISCUSSION THREAD

**Discussion by hholst80** [nushell/nushell#8518] [Q&A]
Why do I need to specify an explicit "get" instead of an implicit get for a .dot prefix string? This would be inline with how jq works.

![image](https://user-images.githubusercontent.com/6200749/226142333-9e170a49-339d-4633-bbb5-2776cc087267.png)

**Comment by fdncred** [comment]
Because, in your third example, you're piping into a cell-path which isn't a command. In nushell, you can do it like the 1st example, or the 2nd example, but not the 3rd way (yet). On the latest main you can also do it like this:
```
let cellpath = $.directories.0.166.directories
du node_modules | get $cellpath
```

There's always a chance to change how it works but I'm not sure if we'd do that or not.
